packages/tupaia-web/src/features/Map/MapOverlaySelector/MapTableModal.tsx

```diff
@@ -101,7 +111,7 @@ export const MapTableModal = ({ setIsOpen }: any) => {
           <TitleWrapper>
             <Title>{titleText}</Title>
             <IconButton>
-              <DownloadIcon />
+              <DownloadIcon onClick={doExport} />
```

Comment:
Can I just chime in here with one request - can you possibly put the onClick handler onto the `IconButton` instead of the `DownloadIcon` so we are more semantic?